app/cdap/components/ConnectionList/styles.ts

```diff
@@ -88,4 +91,37 @@ export const useStyles = makeStyles({
   noDataLabel: {
     paddingLeft: 34,
   },
+  infographContainer: {
+    display: 'flex',
+    flexGrow: 1,
+    justifyContent: 'flex-end',
+    alignItems: ' flex-end',
+    paddingRight: '80px',
+  },
+  infograph: {
+    paddingBottom: '80px',
+  },
+  closeIcon: {
+    '& :hover': {
+      cursor: 'pointer',
+    },
+  },
+  searchBar: {
+    width: '100%',
+    backgroundColor: '#fff',
+    border: 'none',
+    marginLeft: '9px',
+    height: '21px',
+    fontSize: '14px',
+    outline: 0,
+  },
+  afterSearchIconClick: {
+    display: 'flex',
+    backgroundColor: '#fff',
```

Comment:
use MUI colors if possible

Reply:
there is no possible color for this in MUI